stats.c

```diff
@@ -97,6 +97,12 @@ char* onefetch() {
 	return extract_named_version(out, "onefetch ");
 }
 
+char* pfetch() {
+	// NOTE Version in the format "onefetch x.x.x"
```

Comment:
```suggestion
	// NOTE Version in the format "pfetch x.x.x"
```
Oops, missed this one